app/server/src/features/events/questions/resolvers.ts

```diff
@@ -27,21 +27,35 @@ export const resolvers: Resolvers = {
                     node: formattedQuestion,
                     cursor: formattedQuestion.createdAt.getTime().toString(),
                 };
-                // TODO: #QQRedesign delete once code complete
                 ctx.pubsub.publish({
-                    topic: 'questionCRUD',
+                    topic: 'questionCreated',
                     payload: {
-                        questionCRUD: { operationType: 'CREATE', edge } as QuestionOperation,
+                        questionCreated: { edge },
                     },
                 });
+
+                return edge;
+            });
+        },
+        async deleteQuestion(parent, args, ctx, info) {
```

Comment:
Also a potentially better approach would be to add an `isDeleted` field and just add an extra filter when fetching the queue or full question list --  `isDeleted: false`.  The behavior of `deleteQuestion` subscription can remain the same because removing the data from the client is fine.  

I'm thinking in the scenario where we want to delete a question, but want to keep the data (a threat or something of that nature).

Reply:
decided to use the isVisible property that was already there (totally forgot it was there). So it just toggles that and the question list query just filters on that.